README.md

```
@@ -8,13 +8,15 @@
 
-| 프로필 |                    이름                     |    역할    | 담당 기능                                                         |
+## 팀원 구성
+
+| 프로필 |                     이름                     |  역할  | 담당 기능                                                                                     |
 |:---:|:-----------------------------------------:|:--------:|:--------------------------------------------------------------|
-|     | **[김현재](https://github.com/hyunjae3458)** | **null** | **[주요 기능]**<br>- **기능**<br>**[추가/인프라 기능]**<br>- 추가1 <br>- 추가2 |
-|     |   **[문정환](https://github.com/mjohn26)**   | **null** | **[주요 기능]**<br>- **기능**<br>**[추가/인프라 기능]**<br>- 추가1 <br>- 추가2 |
-|     |   **[박승민](https://github.com/raonPsm)**   | **null** | **[주요 기능]**<br>- **기능**<br>**[추가/인프라 기능]**<br>- 추가1 <br>- 추가2 |
-|     |  **[안승리](https://github.com/Atory0206)**  | **null** | **[주요 기능]**<br>- **기능**<br>**[추가/인프라 기능]**<br>- 추가1 <br>- 추가2 |
-|     | **[최종인](https://github.com/jongin-git)**  | **null** | **[주요 기능]**<br>- **기능**<br>**[추가/인프라 기능]**<br>- 추가1 <br>- 추가2 |
+| <img src="https://github.com/hyunjae3458.png" width="100" style="border-radius:50%;"> | **[김현재](https://github.com/hyunjae3458)** | **도서 관리 도메인,<br>README 작성** | **[주요 기능]**<br>- 도서 관리 API 구현 (CRUD 및 커서 페이지네이션)<br>- Naver Book & OCR Space API 연동<br>**[추가/인프라 기능]**<br>- S3 썸네일 업로드 트랜잭션 동기화 |
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)